fsw/nos3_defs/targets.cmake

```diff
@@ -117,7 +117,8 @@ SET(APPLICATION_LIST
         generic_eps
         generic_fss
         generic_reaction_wheel
-	generic_torquer
+        generic_radio
+	    generic_torquer
```

Comment:
Fix spacing.  Tabs should not be used; indentation should be 4 spaces.

Reply:
Resolved in commit 38dd658